aggregator/src/helpers/dotEnvPath.ts

```diff
@@ -3,7 +3,8 @@ import { exists, parseArgs } from "../../deps.ts";
 const args = parseArgs(Deno.args);
 
 export const envName = args.env;
-const dotEnvPath = envName ? `.env.${envName}` : ".env";
+const envNameExists = envName && envName !== "undefined";
```

Comment:
How does the `"undefined"` case occur?

Reply:
When I did not set an `--env` arg, and a string of > 0 length (`"undefined"`) is truthy. Could be a difference in our deno versions?

```
deno 1.15.1 (release, x86_64-unknown-linux-gnu)
v8 9.5.172.19
typescript 4.4.2
```

Reply:
Hmm nah don't think it's a version thing. Just did some experimenting with 1.15.3 and the only way I can get `args.env` to be the *string* `"undefined"` is by running:

```sh
deno run testEnv.ts --env undefined
```

Which, I mean, if you do that, then I suppose resolving `.env.undefined` is what you explicitly asked for.

```ts
// testEnv.ts

import { parse } from "https://deno.land/std@0.103.0/flags/mod.ts";

const args = parse(Deno.args);

console.log(args.env);
```

Reply:
Confirmed, don't know how I was seeing that issue. Reverting.